report_a_breach/templates/form_steps/summary.html

```diff
@@ -148,11 +148,20 @@ <h2 class="govuk-summary-card__title">About the suspected breach</h2>
                         Sanctions regimes breached
                     </dt>
                     <dd class="govuk-summary-list__value">
-                        N/A
+                        {% for regime in sanctions_regimes %}
+                            {{ regime }}
+                            <br>
+                        {% endfor %}
+                        {% if unknown_regime %}
```

Comment:
similarly, we can do 

```suggestion
                        {% if form_data.which_sanctions_regime.unknown_regime %}
```